scripts/archived_scripts/examples_using_smart_components/arbitrage_with_smart_component.py

```diff
@@ -9,7 +9,7 @@
 class ArbitrageWithSmartComponent(ScriptStrategyBase):
     # Parameters
     exchange_pair_1 = ExchangePair(exchange="binance", trading_pair="MATIC-USDT")
-    exchange_pair_2 = ExchangePair(exchange="uniswap_polygon_mainnet", trading_pair="WMATIC-USDT")
+    exchange_pair_2 = ExchangePair(exchange="amm_polygon_mainnet", trading_pair="WMATIC-USDT")
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed